src/pages/home/report/ReportActionItemMessageEdit.js

```diff
@@ -81,7 +81,12 @@ class ReportActionItemMessageEdit extends React.Component {
     updateDraft(newDraft) {
         this.textInput.setNativeProps({text: newDraft});
         this.setState({draft: newDraft});
-        this.debouncedSaveDraft(newDraft);
+
+        if (newDraft.trim().length > 0) {
```

Comment:
Please add a code comment explaining why the two different values are passed to `debouncedSaveDraft` based on the draft length. It will help future developers that come across this to understand the intention.